test/algos/consumers/test_sync_wait.cpp

```diff
@@ -102,7 +102,9 @@ TEST_CASE("sync_wait works if signaled from a different thread", "[consumers][sy
     thread_started = true;
 
     // Wait for a result that is triggered by the impulse scheduler
-    optional<tuple<int>> res = sync_wait(ex::transfer_just(sched, 49));
+    // TODO: find out why this hangs:
+    //optional<tuple<int>> res = sync_wait(ex::transfer_just(sched, 49));
+    optional<tuple<int>> res = sync_wait(ex::on(sched, ex::just(49)));
```

Comment:
I seem to have introduced a regression to ... `schedule_from`, perhaps. This needs investigation. @lucteo if you're interested, perhaps you could have a look.